react/components/Table/index.js

```diff
@@ -81,14 +82,35 @@ class Table extends PureComponent {
     const index = hiddenFields.indexOf(key)
     index === -1 ? newFieldsArray.push(key) : newFieldsArray.splice(index, 1)
     this.setState({ hiddenFields: newFieldsArray })
+
+    const { toolbar } = this.props
+
+    const activeFields = difference(
+      Object.keys(this.props.schema.properties),
+      newFieldsArray
+    )
+
+    toolbar?.fields?.onToggleColumn &&
+      toolbar.fields.onToggleColumn({ toggledField: key, activeFields })
```

Comment:
💡 Just an opinion

Since you're not returning any value, maybe an `if` statement would be better, don't you think?
```suggestion
    if (toolbar?.fields?.onToggleColumn) {
      toolbar.fields.onToggleColumn({ toggledField: key, activeFields })
    }
```
The same applies to other functions